src/test/path-mapping.spec.ts

```diff
@@ -0,0 +1,132 @@
+import { join } from 'path';
+
+import { createExec } from './exec-helpers';
+import {
+  CMD_ESM_LOADER_WITHOUT_PROJECT,
+  CMD_TS_NODE_WITHOUT_PROJECT_FLAG,
+  nodeSupportsEsmHooks,
+  TEST_DIR,
+  installTsNode,
+} from './helpers';
+import { test } from './testlib';
+
+test.beforeAll(installTsNode);
+
+const execBuilder = (
+  command: string,
+  moduleDir: string,
+  tsConfig: string = 'tsconfig.json'
+) => {
```

Comment:
Prefer function declarations in general.

```suggestion
function execBuilder (
  command: string,
  moduleDir: string,
  tsConfig: string = 'tsconfig.json'
) {
```

Reply:
Out of interest, because we often have this discussion on our projects, is this just preference or is one better?

Reply:
Done